node-data/src/message.rs

```diff
@@ -114,20 +114,18 @@ impl Serializable for Message {
         header.topic = buf[0];
 
         let payload = match topic {
-            Topics::NewBlock => {
+            Topics::Candidate => {
                 Payload::NewBlock(Box::new(payload::NewBlock::read(r)?))
             }
-            Topics::FirstReduction | Topics::SecondReduction => {
-                Payload::Reduction(payload::Reduction::read(r)?)
-            }
-            Topics::Agreement => {
-                Payload::Agreement(payload::Agreement::read(r)?)
+            Topics::Validation | Topics::Ratification => {
+                Payload::Validation(payload::Validation::read(r)?)
             }
+            Topics::Quorum => Payload::Quorum(payload::Quorum::read(r)?),
             Topics::Block => Payload::Block(Box::new(ledger::Block::read(r)?)),
             Topics::Tx => {
                 Payload::Transaction(Box::new(ledger::Transaction::read(r)?))
             }
-            Topics::Candidate => Payload::CandidateResp(Box::new(
```

Comment:
Why are we differentiating between Candidate messages and GetCandidateResp ? 
If a request a Candidate I would expect a Candidate message to be received. As far as I can see, the only difference is in the signature, that is, in the GetCandidateResp the generator signature is missing. 
I don't think this is correct

Reply:
`Candidate` (previously NewBlock) is a consensus message that is handled by Consensus loop only while it's running `Proposal` phase. It has also round and step numbers that define when it will be processed.

`GetCandidateResp` is a message that may be received only if `GetCandidate(hash)` msg request is sent.

Reply:
This is very weird in my opinion, but I guess it's not the topic of this PR.
I'll think about it and open an Issue for this if necessary

Reply:
Agreed. I'd rather suggest to get rid of `GetCandidate` -> `GetCandidateResp` message flow, if possible. Let's discuss this offline.